### ISSUE TITLE
Title in repeat includes the word `Insert`

### ISSUE THREAD

**Issue by beatrizserrano** [galaxyproject/galaxy#9658]
Hi,

When I create a repeat with
`<repeat title="Add another image">
</repeat>`

the title in the UI includes already the word `Insert`, so if I end up in a situation like in the screenshot attached (`Insert Add another image`).

Although in most of the cases that is a very helpful feature, sometimes it diminishes the flexibility of the UI. Would it make sense to keep the word `Insert` only when `title` is empty?

Thanks!
Beatriz

<img width="230" alt="insert" src="https://user-images.githubusercontent.com/11300231/80351158-22388480-8872-11ea-8b15-894f993fdadc.png">

**Comment by guerler** [comment]
Not sure if the proper solution here would be to remove the "Add" from the tool xml i.e. use "Another Image" as title instead of "Add Another Image"?

**Comment by beatrizserrano** [comment]
Yes, that's what I ended up doing. But I think that in some cases the word `Insert` might not be the best option. Also, if I am not mistaken, the capitalisation will always be double in the button (i.e. `Insert` and `Another`). That would become another paper cut ;)

**Comment by ksuderman** [comment]
I am looking at this for my first paper-cut and while this is a simple change I can make, I think I agree with @guerler and the tool XML should not include "Include/Add" in its title for two reasons:
1. The title is also used to construct the text for the tooltip ``tooltip: `Add new ${this.options.title} block`,`` which will be more problematic if the title already has the words *Include* or *Add*.
2. Existing tools that specify a title in the `<repeat />` may no longer read correctly as the author most likely assumed a leading *Insert*.

**Comment by dannon** [comment]
Should we go a little further and just have the repeat title be *only* the noun in most cases?  With "Insert" and "Insert another" programmatically generated?

**Comment by ksuderman** [comment]
I think recommending just nouns in the title make sense.  I did a quick `find | grep` of existing tool XML files and that is what everyone does currently.  Is it worth the effort to add the logic to toggle the button label between *Insert* and *Insert another* when parameters are added/removed?
